packages/main/test/pages/Table.html

```diff
@@ -43,7 +43,7 @@
 			<ui5-label>Supplier</ui5-label>
 		</ui5-table-column>
 
-		<ui5-table-column id="column-3" slot="columns" min-width="600" popin-text="Dimensions" demand-popin>
+		<ui5-table-column id="column-3" slot="columns" min-width="600" popin-text="Dimensions" demand-popin popin-display="Inline">
```

Comment:
![Screenshot 2022-11-14 at 10 47 51](https://user-images.githubusercontent.com/11913394/201616135-5975809b-1b97-40b5-b170-cfd94c159d12.png)
I think that the alignment is not okay. Could you please check?

Reply:
Fixed for all cells inline.